src/private/patient/js/patientDashboardAPI.js

```diff
@@ -0,0 +1,23 @@
+document.addEventListener('DOMContentLoaded', function () {
+});
+
+welcomeUser = document.getElementById('welcomeUser');
+
+
+window.onload = function writeWelcomeAndUserName() {
```

Comment:
Tylko jedna rzecz chcemy zrobić po wczytaniu okna ?

Reply:
A na stronie głównej potrzeba czegoś więcej?

Reply:
np wczytanie oczekujacych wizyt?

Reply:
ok, myślałam, żeby to dodać do "twoje wizyty", ale w sumie spoko pomysł